libnetwork/network.go

```diff
@@ -1864,6 +1864,7 @@ func (n *Network) Ingress() bool {
 	return n.ingress
 }
 
+// Dynamic indicates whether this network was created on-demand by Swarm
 func (n *Network) Dynamic() bool {
```

Comment:
Perhaps describe some of the behaviour of dynamic networks (could be useful); but check if my wording is correct (on the "create / remove" behaviour 😅 

```suggestion
// Dynamic indicates whether thisNetwork is a dynamic (Swarm-managed) network.
// Dynamic networks are rolled out dynamically by the Swarm when tasks are
// deployed to a node in the cluster. Similarly, they are removed when there's
// no more tasks on the node that use the network.
//
// Dynamic networks may be "attachable", which allows non-Swarm ("non-managed")
// containers to attach to them if the network is present on a node.
func (n *Network) Dynamic() bool {
```